src/spetlrtools/diagrams/Edge.py

```diff
@@ -0,0 +1,10 @@
+from collections import namedtuple
+
+_Edge = namedtuple("_Edge", "source target")
+
+
+class Edge(_Edge):
+    def __new__(cls, source: str, target: str, *, style=None):
```

Comment:
add return type

Reply:
__new__ returns an uninitalized blank object that is fed into __init__. I don't think it needs a return type since it is never used expicitly.